Pod/Core/XCTestCase+Gherkin.swift

```diff
@@ -17,7 +17,7 @@ UITestCase - a subclass wouldn't work with both of them.
 It's nicer code IMHO to have the state as a single associated property beacuse of the grossness of setting/getting it.
 This means that anytime I want to access my extra properties I just do `state.{propertyName}`
 */
-private class GherkinState {
+public class GherkinState {
```

Comment:
Hey.

I’ve just pushed version 0.5.0 live so you should be able to do a pod update and get your changes now :)

Thanks,
S

From: "@smaljaars" <notifications@github.com>
Reply-To: net-a-porter-mobile/XCTest-Gherkin <reply@reply.github.com>
Date: Saturday, June 18, 2016 at 7:26 AM
To: net-a-porter-mobile/XCTest-Gherkin <XCTest-Gherkin@noreply.github.com>
Cc: Sam Dean <sam.dean@net-a-porter.com>, Comment <comment@noreply.github.com>
Subject: Re: [net-a-porter-mobile/XCTest-Gherkin] Features/gherkin step checker (#24)


In Pod/Core/XCTestCase+Gherkin.swift<https://github.com/net-a-porter-mobile/XCTest-Gherkin/pull/24#discussion_r67596739>:

> @@ -17,7 +17,7 @@ UITestCase - a subclass wouldn't work with both of them.

>  It's nicer code IMHO to have the state as a single associated property beacuse of the grossness of setting/getting it.

>  This means that anytime I want to access my extra properties I just do `state.{propertyName}`

>  */

> -private class GherkinState {

> +public class GherkinState {

yes that's true, we don't need public, I changed it. The GherkinState is only accessed within the XCTest-Gherkin project itself.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/net-a-porter-mobile/XCTest-Gherkin/pull/24/files/ccb26306ca109456ff97e33cf1ecbef0348367a1#r67596739>, or mute the thread<https://github.com/notifications/unsubscribe/ADRNEuLX_DpeRAWuX4tZzCDlw8qlBBynks5qM4-HgaJpZM4I4LTI>.


NET-A-PORTER.COM



CONFIDENTIALITY NOTICE
The information in this email is confidential and is intended solely for the addressee. Access to this email by anyone else is unauthorised. If you are not the intended recipient, you must not read, use or disseminate the information. Any views expressed in this message are those of the individual sender, except where the sender specifically states them to be the views of Net-A-Porter Group Limited. 

The Net-A-Porter Group Limited is a company registered in England & Wales Number: 3820604 Registered Office: 1 The Village Offices, Westfield, Ariel Way, London, W12 7GF